src/main/webapp/src/components/Basket.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import axios from "axios";
+import {Button} from "antd";
+import {DeleteFilled, ShoppingOutlined} from "@ant-design/icons";
+import '../styles/Basket.css'
+import '../styles/NotFoundPage.css'
+import {Link} from "react-router-dom";
+
+class Basket extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            basket: null,
+            phones: [],
+            numberItems: 0,
+            totalPrice: 0,
```

Comment:
why do you have these fields? it is already available in basket field if it is not null

Reply:
+